chore: streamline acctests setup (#1847)
hashicorp/terraform-provider-vsphere: acctests/README.md

6. Download the vCenter Server appliance ISO image.

## Extract
Next you need to mount and extract the contents of the iso to a location on your disk.

### macOS Users

> During the Equinix provisioning, macOS users may get security issues when ovftool is attempting to run, to prevent this issue please run:

Reviewer: ```suggestion
During the Equinix provisioning, macOS users may get security issues when ovftool is attempting to run, to prevent this issue run the following command:
```

export TF_VAR_VSPHERE_LICENSE="XXXXX-XXXXX-XXXXX-XXXXX-XXXXX"
# set this to the location of the extracted vsca installer for your OS
export TF_VAR_VCSA_DEPLOY_PATH="{extract_location}/vcsa-cli-installer/mac/vcsa-deploy"
```

## Provision Equinix Infrastructure

> Terraform needs to be ran in 2 stages, once to provision Equinix infrastructure, and a second time to create some shared resources most acceptances tests rely on within vSphere.

Reviewer: ```suggestion
Terraform needs to be run in two stages. First, to provision the Equinix infrastructure and second to create the shared resources upon which most acceptance tests rely.
```

```

## Run Terraform Phase 2

> A file with several environment variables called `devrc` should have been created within `acctests/equinix`. You should source that file and then switch to `acctests/vsphere`.

Reviewer: ```suggestion
A file with several environment variables called `devrc` must be created within `acctests/equinix` path. You must source that file and then switch to the `acctests/vsphere` path.
```

```
$ source devrc
$ cd ../vsphere
$ terraform init
$ terraform apply
```

> Now all the infrastructure should be created, if you planned on running the cluster tests with vSAN, make sure to enable vSAN in VMKernal Adapters on the management network for ESXI hosts 3 and 4. This is must be done through the web console. Another `devrc` file should have been created within the `acctests/vsphere` folder, source that now and you should have all the necessary environment variables for running tests.

Reviewer: ```suggestion
Now all the infrastructure should be created, if you planned on running the cluster tests with vSAN, make sure to enable vSAN on the VMkernal adapters on the management network for ESXI hosts 3 and 4. This can be done through the vSphere client or PowerCLI.  An additional `devrc` file should have been created in the `acctests/vsphere` path. Source this file and you should have all the necessary environment variables for running tests.
```
